test/dynamo/test_python_dispatcher.py

```diff
@@ -0,0 +1,84 @@
+# Owner(s): ["module: dynamo"]
+import unittest
+
+import torch
+import torch._dynamo.test_case
+from torch._dynamo.testing import CompileCounter, EagerAndRecordGraphs, normalize_gm
+from torch.testing._internal.common_cuda import TEST_CUDA
+
+
+class PythonDispatcherTests(torch._dynamo.test_case.TestCase):
+    def test_dispatch_key1(self):
+        @torch.compile(backend="aot_eager", fullgraph=True)
+        def fn(x):
+            x = x + 1
+            return torch._C._dispatch_keys(x)
+
+        x = torch.randn(2, 3)
+        self.assertTrue(fn(x).raw_repr() == torch._C._dispatch_keys(x + 1).raw_repr())
+
+    def test_dispatch_key2(self):
+        eager = EagerAndRecordGraphs()
+
+        @torch.compile(backend=eager, fullgraph=True)
+        def fn(x):
+            key_set = torch._C._dispatch_tls_local_include_set()
```

Comment:
Can we return the result of `_dispatch_tls_local_include_set` in a test and compare it between eager and compile?